matter_server/client/client.py

```diff
@@ -94,11 +99,11 @@ def unsubscribe() -> None:
 
         return unsubscribe
 
-    async def get_nodes(self) -> list[MatterNode]:
+    def get_nodes(self) -> list[MatterNode]:
```

Comment:
These method changes on the client are breaking. We should note that in the PR description and tag the PR.